README.md

```diff
@@ -50,3 +50,164 @@ FAQ
+          "elaostrap.fancybox": "./node_modules/elaostrap/assets/js/vendors/jquery.fancybox.pack.js",
+          "elaostrap.easing": "./node_modules/elaostrap/assets/js/vendors/jquery.easing.js",
+          "elaostrap.mobileNav": "./node_modules/elaostrap/assets/js/vendors/mobile-nav.js"
+      },
+      "browserify-shim": {
+          "bootstrap.transition": {
+              "depends": "jquery:jQuery"
+          },
+          "bootstrap.collapse": {
+              "depends": [
+                  "jquery:jQuery",
+                  "bootstrap.transition"
+              ]
+          },
+          "bootstrap.tab": {
+              "depends": "jquery:jQuery"
+          },
+          "bootstrap.tooltip": {
+              "depends": "jquery:jQuery"
+          },
+          "bootstrap.popover": {
+              "depends": "jquery:jQuery"
+          },
+          "bootstrap.dropdown": {
+              "depends": "jquery:jQuery"
+          },
+          "dropify": {
+            "depends": "jquery:jQuery"
+          }
+      },
+      "scripts": {
+          "install": "napa"
+      },
+      "browserify": {
+          "transform": ["browserify-shim"]
+      },
+      "devDependencies": {
+          "gulp": "3.8.*",
+          "del": "1.1.*",
+          "elao-assets-gulp": "~0.1.16",
+          "napa": "1.1.*",
+          "browserify-shim": "3.8.*"
+      }
+    }
+    
+Update your /gulpfile.js, here an example :
+    
+    assets.config({
+        (...),
+        
+        assets: {
+            fonts: {
+                groups: {
+                    'elaostrap': {src: 'elaostrap/assets/fonts/**'},
+                    'font-awesome': {src: 'font-awesome/fonts/**', dest: 'font-awesome'},
+                    'dropify': {src: 'dropify/src/fonts/**', dest: 'dropify'},
+                }
+            }
+        }
+    });
+    
+Then do in your Vagrant VM :
+
+    $ npm install
+    
+An example how to call the assets in your app/Resources/views/base.html.twig :
+    
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <title>{% block title %}My title{% endblock %}</title>
+            <meta name="description" content="">
+            <meta charset="utf-8">
+            <meta http-equiv="X-UA-Compatible" content="IE=edge">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
+            <meta name="format-detection" content="telephone=no">
+            {% block stylesheets %}
+                <link rel="stylesheet" href="{{ asset('assets/css/web/main.css') }}">
+            {% endblock %}
+        </head>
+        <body>
+            {% block body %}{% endblock %}
+    
+            {% block javascripts %}
+                <script src="{{ asset('assets/js/web/main.js') }}"></script>
+            {% endblock %}
+        </body>
+    </html>
+
+If your Bundle is AppBundle where you store your assets, here an example of files structure and some code :
```

Comment:
You can even say something like: "Here is an example of files structure and code, considering the bundle where you store your assets is called AppBundle:"